PSReadLine/BasicEditing.cs

```diff
@@ -178,9 +169,7 @@ private void DeleteCharImpl(int qty, bool orExit)
                 {
                     qty = Math.Min(qty, _singleton._buffer.Length - _singleton._current);
 
-                    SaveEditItem(EditItemDelete.Create(_buffer.ToString(_current, qty), _current, DeleteChar, qty));
-                    SaveToClipboard(_current, qty);
-                    _buffer.Remove(_current, qty);
+                    RemoveTextToClipboard(_current, qty, DeleteChar);
```

Comment:
The original `EditItemDelete.Create(_buffer.ToString(_current, qty), _current, DeleteChar, qty)` line passed `qty` as the `arg`, so I guess we should do the same here:
```suggestion
                    RemoveTextToClipboard(_current, qty, DeleteChar, qty);
```

Reply:
Done.